content/en/docs/reference/labels-annotations-taints/_index.md

```diff
@@ -1133,3 +1133,6 @@ Example: `node-role.kubernetes.io/master:NoSchedule`
 Taint that kubeadm previously applied on control plane nodes to allow only critical workloads to schedule on them.
 Replaced by [`node-role.kubernetes.io/control-plane`](#node-role-kubernetes-io-control-plane-taint); kubeadm
 no longer sets or uses this deprecated taint.
+{{< note >}}
+The EndpointSlice controller ignores nodes with the `node-role.kubernetes.io/control-plane` or `node-role.kubernetes.io/master` label set.
+{{< /note >}}
```

Comment:
Is this true?

Reply:
Hi @reylejano, @robscott 
Can you please comment on this.